main.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import os
+from datetime import datetime
+
+import schedule
+
+parser = argparse.ArgumentParser("negotiator_directory_sync_arguments")
+parser.add_argument("--directory_emx2_endpoint", help="The url of the Directory endpoint (EMX2)", type=str)
+parser.add_argument("--negotiator_endpoint", help="The url of the Negotiator API endpoint (EMX2)", type=str)
+parser.add_argument("--negotiator_client_auth_client_id", help="The auth client ID for Negotiator authentication",
+                    type=str)
+parser.add_argument("--negotiator_client_auth_client_secret",
+                    help="The auth client secret for Negotiator authentication", type=str)
+parser.add_argument("--negotiator_client_auth_oidc_token_endpoint", help="The endpoint for Negotiator authentication",
+                    type=str)
+parser.add_argument("--sync_job_schedule_interval", help="The interval for sync job execution (in seconds)", type=int)
+args = parser.parse_args()
+os.environ['DIRECTORY_API_URL'] = args.directory_emx2_endpoint
+os.environ['NEGOTIATOR_API_URL'] = args.negotiator_endpoint
+os.environ['AUTH_CLIENT_ID'] = args.negotiator_client_auth_client_id
+os.environ['AUTH_CLIENT_SECRET'] = args.negotiator_client_auth_client_secret
+os.environ['AUTH_OIDC_TOKEN_URI'] = args.negotiator_client_auth_oidc_token_endpoint
+os.environ['JOB_SCHEDULE_INTERVAL'] = str(args.sync_job_schedule_interval)
```

Comment:
We should probably move this to a separate class

Reply:
Fixed in [4df2a93](https://github.com/BBMRI-ERIC/directory-negotiator-sync/pull/1/commits/4df2a934ebed4a56a823da4d2fbaec1eb68ef804)